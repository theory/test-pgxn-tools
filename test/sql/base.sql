\i widget.sql
SELECT widget();
