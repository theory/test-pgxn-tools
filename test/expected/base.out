\i widget.sql
SET client_min_messages = warning;
CREATE OR REPLACE FUNCTION widget()
RETURNS TEXT LANGUAGE SQL AS 'SELECT ''widget''::TEXT';
SELECT widget();
 widget 
--------
 widget
(1 row)

